tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "noEmit": true,
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
I'd like this to be true. What types are any when this is enabled?

Reply:
Actually, this should be `true`. No types should default to `any`